ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/package.scala

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.ledger.participant.state
+package com.daml
```

Comment:
Please revert this change. We don't use this package notation in `kvutils`.

Reply:
I revert this change in  #9433